src/astx/transpilers/python.py

```diff
@@ -282,3 +282,8 @@ def visit(self, node: astx.LiteralUTF8String) -> str:
     def visit(self, node: astx.LiteralUTF8Char) -> str:
         """Handle LiteralUTF8Char nodes."""
         return repr(node.value)
+
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.LiteralBoolean) -> str:
+        """Handle LiteralBoolean nodes."""
+        return "True" if node.value else "False"
```

Comment:
Assuming that the value should be a boolean value, this should resolve this in a shorter way

```suggestion
        return repr(node.value)
```

Reply:
but this is not a block for this PR, let's merge it and we can address that later